src/ragas/metrics/__init__.py

```diff
@@ -15,6 +17,37 @@
 from ragas.metrics._context_relevancy import ContextRelevancy, context_relevancy
 from ragas.metrics._faithfulness import Faithfulness, faithfulness
 from ragas.metrics.critique import AspectCritique
+from ragas.metrics.base import Metric
+from ragas.validation import validate_evaluation_modes
+
+ALL_METRICS = [
+    answer_correctness,
+    faithfulness,
+    answer_similarity,
+    context_precision,
+    context_utilization,
+    context_recall,
+    context_relevancy,
+    answer_relevancy,
+    context_entity_recall,
+]
+
+
+def get_available_metrics(ds: Dataset) -> list[Metric]:
```

Comment:
could you move this to a new file called `metrics/utils` and import it here?
that will be much cleaner

Reply:
yeah, my plan was to do in utils.py only, but it was having circular dependency. Raised new changes which brings this function to `metrics/utils`